components/event-bus/cmd/event-publish-service/publisher/publisher.go

```diff
@@ -133,7 +125,17 @@ func (publisher *DefaultKnativePublisher) publishOnChannel(knativeLib *knative.K
 		log.Printf("failed to send message to the knative channel '%v' in namespace '%v'", channel.Name, *namespace)
 		return api.ErrorResponseInternalServer(), FAILED, channel.Name
 	}
+	report(channel, PUBLISHED, *namespace)
 
 	// publish to channel succeeded return nil error
 	return nil, PUBLISHED, channel.Name
 }
+
+func report(channel *messagingV1Alpha1.Channel, status, ns string) {
```

Comment:
<nitpick>
something similar to `updateMetrics` would be more descriptive
</nitpick>

Reply:
Done